docs/architecture.md

```diff
@@ -160,17 +160,14 @@ with `spec.replica` set to `1`.
 ## How to use a VirtualMachine
 
 A VirtualMachine will make sure that a VirtualMachineInstance object
-with an identical name will be present in the cluster, if `spec.running`
-is set to `true`. Further it will make sure that a
+with an identical name will be present in the cluster, if `spec.runStrategy`
+is set to `Always`. Further it will make sure that a
 VirtualMachineInstance will be removed from the cluster if
-`spec.running` is set to `false`.
+`spec.runStrategy` is set to `Halted`.
 
-There exists a field `spec.runStrategy` which can also be used to
-control the state of the associated VirtualMachineInstance object. To
-avoid confusing and contradictory states, these fields are mutually
-exclusive.
-
-An extended explanation of `spec.runStrategy` vs `spec.running` can be
+`spec.runStrategy` has several additional options for
+controlling the state of the associated VirtualMachineInstance object.
+An extended explanation of `spec.runStrategy` can be
```

Comment:
Looks good to me. An alternative to this explanation could be something like:
```
A VirtualMachine will make sure that a VirtualMachineInstance object
with an identical name will be present in the cluster when the VirtualMachine
is in a Running state, which is controlled via the `spec.runStrategy` field.
For more information regarding Run Strategies, please refer to [Run Strategies](./compute/run_strategies.md).
```

The rationale is to avoid diving into the different strategies here, and just outline the main idea and refer to the right place for more info.

WDYT?

Reply:
Much better!